index.html

```diff
@@ -129,7 +129,7 @@ <h1 class="title ellipsis" ng-show="$root.iden && $root.wallet  && !$root.hideWa
           ng-class="{'main':$root.iden && !$root.starting && !$root.hideNavigation  && !$root.wpInputFocused}" 
           ng-view></section> 
 
-        <a class="exit-off-canvas"></a>
+        <a class="close-menu" ng-click="swipe()"></a>
```

Comment:
we should decide whether wi will call swipe() functions or openMenu() functions on the HTML. It does not make sense to call swipe() and openMenu(). It should be `openMenu()` and `closeMenu()`.